jira/resources.py

```diff
@@ -911,6 +911,11 @@ def __init__(self, options, session, raw=None):
 
 # Utilities
 
+def setattrWithUnicode(obj, name, value):
+    try:
+        setattr(obj, name, value)
+    except Exception:
```

Comment:
You should only catch specific exceptions you've planned for. When an unexpected exception happens your program _should_ stop, rather than proceeding blindly forward, which makes catching `Exception` undesirable behaviour.